dnf-repoconfig-daemon/dbus/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+SET(SYSCONFDIR /etc)
+SET(SYSTEMD_UNIT_DIR /usr/lib/systemd/system)
+SET(DBUS_SERVICES_DIR /usr/share/dbus-1/system-services)
+SET(DBUS_CONFIG_DIR ${SYSCONFDIR}/dbus-1/system.d)
```

Comment:
should be lower case
check https://github.com/dmach/cmake-tidy for automated fix